src/routes/webhooks/topgg.ts

```diff
@@ -0,0 +1,37 @@
+import ApiRequest from '@lib/structures/api/ApiRequest';
+import ApiResponse from '@lib/structures/api/ApiResponse';
+import { Route, RouteOptions } from 'klasa-dashboard-hooks';
+import { TOKENS } from '@root/config';
+import { UserSettings } from '@lib/types/settings/UserSettings';
+import { ApplyOptions } from '@skyra/decorators';
+
+@ApplyOptions<RouteOptions>({ name: 'webhooks/topgg', route: 'webhooks/topgg' })
+export default class extends Route {
+
+	public async post(request: ApiRequest, response: ApiResponse) {
+		if (request.headers.authorization !== TOKENS.WEBHOOK_B4D) return response.forbidden();
+		if (!request.body) return response.badRequest();
+
+		const body = request.body as Body;
+		try {
+			const user = await this.client.users.fetch(body.user);
+			const settings = await user.settings.sync();
+			const payment = body.isWeekend ? 600 : 400;
```

Comment:
```suggestion
			const payment = body.isWeekend ? 800 : 400;
```

In the weekend user's votes count as 2 so we should also give them twice the normal amount of shinies IMO